rcsbapi/sequence/seq_query.py

```diff
@@ -110,36 +118,22 @@ class Alignments(Query):
     """
     Get sequence alignments
 
-        from_ (str): From which query sequence database
-        to (str): To which query sequence database
-        queryId (str): Sequence identifier specified in `from_`
+        db_from (str): From which query sequence database
+        db_to (str): To which query sequence database
+        query_id (str): Sequence identifier specified in `db_from`
         return_data_list (List[str]): Fields to request data for
         range (Optional, List[]): Optional integer list to filter annotations that fall in a particular region
         suppress_autocomplete_warning (bool, optional): Suppress warning message about field path autocompletion. Defaults to False.
         _query (MappingProxyType): Attribute for storing GraphQL query
```

Comment:
Can you add `data_list_args` to this list along with a description (including mention that `offset` and `first` are currently accepted)?